meta/main.yml

```diff
@@ -3,15 +3,39 @@ galaxy_info:
   author: Shane Frasier
   description: Install MongoDB.
   company: CISA Cyber Assessments
+  galaxy_tags:
+    - mongodb
+    - mongo
   license: CC0
-  min_ansible_version: 2.0
+  # With the release of version 2.10, Ansible finally correctly
+  # identifies Kali Linux as being the Kali distribution of the Debian
+  # OS family.  This simplifies a lot of things for roles that support
+  # Kali Linux, so it makes sense to force the installation of Ansible
+  # 2.10 or newer.
+  min_ansible_version: 2.10
+  namespace: cisagov
   platforms:
+    # - name: Amazon
+    #   versions:
+    #     - 2
     - name: Debian
       versions:
         - stretch
-  galaxy_tags:
-    - mongodb
-    - mongo
+    #     - buster
+    #     - bullseye
```

Comment:
Is there a reason we aren't including buster and bullseye now?  I also don't remember why we excluded the other platforms here as well, but I'm less concerned about them.

Reply:
This Ansible role [only ever supported Stretch](../blob/e8e178464576e7b54b51e9047eb2d9acaed0dfb5/files/mongodb-org-3.6.list#L1).  It probably does need to be expanded to support Buster (at a minimum) but I think that should be a separate PR.

Reply:
That works for me.  Can you please create an issue to support Buster (and beyond?) so we don't forget about it?

Reply:
Thank you sir!